### ISSUE TITLE
Bad python programming practices in variaveis.py (soon to be renamed variables.py)

### ISSUE THREAD

**Issue by gabriel-libardi** [lucasgpulcinelli/ICMC-assembly-preprocessor#6]
The script should a main function, and the execution should be controlled by an `if __name__ ==  '__main__'` block. Furthermore, procedures from the code should be separated in functions, and be properly documented. The code in variaveis.py also has inconsistent style with the code style of preprocessor.py.

**Comment by lucasgpulcinelli** [comment]
I Agree there are many bad practices in this file, could you work with @matheuspd, the original creator of the script, to fix this?
